src/message-fragmentation.md

```diff
@@ -6,16 +6,22 @@ destination node is responsible for collecting the fragments together and reasse
 
 ## Fragment structure
 
+// TODO it is not exactly clear where this fragment is when looking at `sphinx`. IIUC sphinx payload contains
+// the fragment. (means that some of the point I raised in `sphinx.md` can just refer to this md.
```

Comment:
Yes, the Sphinx payload contains a fragment in the case of request and reply packets. This is covered in the Sphinx chapter: "After decryption/encryption of a request/reply payload, if the last 16 bytes of the payload are 0, the rest of the payload should be handled as a message fragment"